lib/beaker/platform.rb

```diff
@@ -120,7 +120,7 @@ def base_packages
       when 'el'
         @version.to_i >= 8 ? ['curl-minimal', 'iputils'] : %w[curl]
       when 'debian'
-        %w[curl lsb-release apt-transport-https]
+        %w[curl lsb-release]
```

Comment:
Unrelated, but do we still need `lsb-release`? In the past it was needed for proper facts on Debian but I think current Facter uses `/etc/os-release` to determine the values.

Reply:
Made #1904 to track this.